README.md

```diff
@@ -249,25 +249,23 @@ The request parameters are much more limited and only travel time is returned. I
 This inflexibility comes with a benefit of faster response times (Over 5x faster compared to regular time filter) and larger limits on the amount of destination points.
 
 Body attributes:
-* origin: Origin point.
-* destination: Destination points. Cannot be more than 200,000.
-* transportation: Transportation type.
+* origin: Origin point;
+* destination: Destination points. Cannot be more than 200,000;
+* transportation: Transportation type;
 * travelTime: Time limit;
-* country: Return the results that are within the specified country
+* country: Return the results that are within the specified country;
+* requestType: MANY_TO_ONE(single departure location and multiple arrival locations) or ONE_TO_MANY (single arrival location and multiple departure locations).
```

Comment:
This is mixed up.

one-to-many -> single departure location and multiple arrival locations
many-to-one -> single arrival location and multiple departure locations